modules/ols-creating-the-credentials-secret-using-cli.adoc

```diff
@@ -42,52 +42,56 @@ stringData:
 ----
 apiVersion: v1
 data:
-  apitoken: <api_token>
+  apitoken: <api_token> <1>
 kind: Secret
 metadata:
   name: rhelai-api-keys
   namespace: openshift-lightspeed
 type: Opaque
 ----
+<1> The `apitoken` must be `base64` encoded when stored in a secret.
```

Comment:
apitoken -> its called as `api_token> <1>` - should these be `api_token` ?

Reply:
done